src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -621,13 +685,17 @@ private void generateDuplicateIndexes(final boolean useBarcodes, final boolean i
             } else {
                 if (nextChunk.size() > 1) {
                     markDuplicatePairs(nextChunk);
+                    if (TAG_DUPLICATE_SET_MEMBERS) addRepresentativeReadIndex(nextChunk);
```

Comment:
My thought is that since it is only repeated once and is only 2 lines, that abstracting it would make it a little less readable and one wouldn't gain too much having a new function. Not a big deal to me either way though!